packages/canary-react/src/stories/ic-select_(multi).stories.mdx

```diff
@@ -0,0 +1,896 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+} from "@storybook/addon-docs";
+import { IcSelectWithMulti } from "../components";
+import { useRef } from "react";
+import readme from "../../../canary-web-components/src/components/ic-select-with-multi/readme.md";
+import {
+  OPTIONS,
+  OPTIONS_WITH_DESCRIPTIONS,
+  OPTIONS_WITH_DISABLED,
+  GROUPED_OPTIONS,
+  OPTIONS_WITH_RECOMMENDED,
+} from "../../../canary-web-components/src/components/ic-select-with-multi/story-data";
+
+<Meta
+  title="React Components/Multi-Select"
+  component={IcSelectWithMulti}
+  parameters={{
+    componentAPI: {
+      data: readme,
+    },
+  }}
+/>
+
+# [Canary]: Multi-Select
+
+Use the multi-select component to allow users to select one or more values from a list of options.
+
+There is one required prop for the multi-select:
+
+- label: `string`
+
+Click the 'Component API' tab to view all the available props, events and slots for the multi-select.
+
+To use the multi-select component, import `@ukic/canary-react` into your application.
+
+The component tag name for the multi-select is `IcSelectWithMulti`. 
+Even though it is a variant of the `IcSelect` component (which includes the single and searchable selects), this different tag name is necessary because of the need to keep the multi-select functionality separate while it is experimental (in a canary release).
+Note that the `multiple` prop still needs to be applied in order to enable the multi-select functionality.
```

Comment:
I think one of the main reasons I did this is so that it's most consistent with how the multi-select functionality will be enabled once it has been released in the main package(s). 

When it is released, it would require a bit of work for us to change to using the `multiple` prop rather than the component tag name to determine whether to enable the multi-select functionality. And it may be a little confusing for customers, having to change their code to add `multiple` props that they didn't have before (alongside the change to use the `ic-select` tag).

I felt it was more efficient and easier to just use the `multiple` prop from the start.

Also, I feel like using just the tag name to determine whether to enable the multi-select functionality may complicate things if, at any point, we want to check that the behaviour of the single and searchable variants of the `ic-select-with-multi` component still works correctly (such as how I had the single and searchable stories in [my first PR](https://github.com/mi6/ic-ui-kit/pull/1385) for the purpose of checking that I hadn't introduced bugs / affected existing behaviour).

Reply:
Valid points. Sounds good.